sat-plugin/src/test/java/org/openhab/tools/analysis/report/test/ReportUtilityTest.java

```diff
@@ -107,7 +107,7 @@ public void assertWarningAreLoggedWhileExecuting() throws MojoFailureException {
         verify(logger).error("org.eclipse.smarthome.auth.jaas.internal.JaasAuthenticationProvider.java:[69]\nComment matches to-do format '(TODO)|(FIXME)'.");

```

Comment:
Could we remove all `org.eclipse.smarthome` references as part of this PR as well?